spec/Flagbit/Bundle/CategoryBundle/Transformer/Denormalization/BooleanTransformerSpec.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+declare(strict_types=1);
+
+namespace spec\Flagbit\Bundle\CategoryBundle\Transformer\Denormalization;
+
+use Flagbit\Bundle\CategoryBundle\Transformer\Denormalization\BooleanTransformer;
+use PhpSpec\ObjectBehavior;
+
+/**
+ * @method transform(mixed $data)
+ */
+class BooleanTransformerSpec extends ObjectBehavior
+{
+    public function it_is_initializable(): void
+    {
+        $this->shouldHaveType(BooleanTransformer::class);
+    }
+
+    public function it_returns_one_with_boolean_true_data_argument(): void
+    {
+        $this->transform(true)->shouldReturn(1);
+    }
+
+    public function it_returns_zero_with_boolean_false_data_argument(): void
+    {
+        $this->transform(false)->shouldReturn(0);
+    }
+
+    public function it_returns_one_with_int_one_data_argument(): void
+    {
+        $this->transform(1)->shouldReturn(1);
+    }
+
+    public function it_returns_zero_with_int_zero_data_argument(): void
+    {
+        $this->transform(0)->shouldReturn(0);
+    }
```

Comment:
I don't expect booloean or integer from an import file, but I think we can keep them for completeness.